src/coreComponents/physicsSolvers/wavePropagation/pygeosx/receiver.py

```diff
@@ -0,0 +1,121 @@
+'''
+Created on 10/02/2021
+
+@author: macpro
+'''
+import numpy as np
+
+class Receiver:
+    """A class representing a receiver
+    
+    Attributes
+    ----------
+    coords :
+        Coordinates of the source
+    """
+    
+    def __init__(self,pos):
+        """Constructor for the receiver
+        
+        Parameters
+        ----------
+        pos :
+            Coordinates for the receiver
+        """
+        
+        self.x = pos[0]
+        self.y = pos[1]
+        self.z = pos[2]
```

Comment:
Get rid of these. That will simplify `setReceiverPos` and `linearMove`, which don't currently update `self.coords`.

Reply:
All of these have been removed now